modules/ROOT/pages/dw-dataformat-types.adoc

```diff
@@ -1,61 +1,53 @@
 = DataFormat Types (dw::extension::DataFormat)
 
+The DataFormat module provides resources for registering a new data format for the DataWeave language. For an example, see
+https://github.com/mulesoft-labs/data-weave-custom-data-format[Custom Data Formats Example].
+
+[%header, cols="1,2a,3a"]
 |===
 | Type | Definition | Description
 
 | DataFormat
-| `type DataFormat = { /**
-* True if this is data format is represented in a binary representation instead of text, if not present is false
-**/
-binaryFormat?: Boolean, /**
-* The default charset of this format, if any.
-**/
-defaultCharset?: String, /**
-* Returns the list of file extensions with the . &#40;".json", ".xml", etc...&#41; that should be assigned to this Data Format
-**/
-fileExtensions?: Array<String&#62;, /**
-* The list of MimeTypes that are accepted
-**/
-acceptedMimeTypes: Array<MimeType&#62;, /**
-* This function will be in charge of reading the raw content and transform it into the DW canonical model
-**/
-reader: &#40;content: Binary, charset: String, settings: ReaderSettings&#41; &#45;&#62; Any, /**
-* This function will be in charge of writing the DW canonical model into Binary content
-**/
-writer: &#40;value: Any, settings: WriterSettings&#41; &#45;&#62; Binary }`
-| Represents the DataFormat definition.
-
-_Introduced in DataWeave: 2.2.0._
+| `type DataFormat = { binaryFormat?: Boolean, defaultCharset?: String,
```

Comment:
TODO (DUKE): consider new format (bullets?) that is easier for doc builder to produce without errors

Reply:
TODO: (DUKE): revise descriptions in DWL source